dlk/python/dlk/templates/src/func/pad.cpp

```diff
@@ -0,0 +1,38 @@
+#include "global.h"
+#include "func/pad.h"
+#include "time_measurement.h"
+
+void func_Pad(T_FLOAT input[], int32_t padding[], T_FLOAT output[],
+              T_UINT in_depth, T_UINT out_height, T_UINT out_width, T_UINT out_depth) {
+#ifndef RUN_AS_HLS
+  Measurement::Start("Pad");
+#endif
+
+  T_UINT elements = out_height * out_width * out_depth;
+  T_UINT out_idx = 0;
+  T_UINT in_idx = 0;
+
+  T_UINT prep = padding[6];
```

Comment:
???
It looks be able to apply only channel-wise zero padding.
Should write document the